2017-12-01 09:13:01,033 handlers.py[line:104] ERROR NoneType

2017-12-01 09:14:01,759 handlers.py[line:104] ERROR NoneType

2017-12-01 09:15:02,406 handlers.py[line:104] ERROR NoneType

2017-12-01 09:15:02,408 handlers.py[line:104] ERROR NoneType

2017-12-01 09:16:03,096 handlers.py[line:104] ERROR NoneType

2017-12-01 09:16:03,104 handlers.py[line:104] ERROR NoneType

2017-12-01 09:17:03,781 handlers.py[line:104] ERROR NoneType

2017-12-01 09:18:04,533 handlers.py[line:104] ERROR NoneType

2017-12-01 09:19:05,199 handlers.py[line:104] ERROR NoneType

2017-12-01 09:19:05,203 handlers.py[line:104] ERROR NoneType

2017-12-01 09:20:05,912 handlers.py[line:104] ERROR NoneType

2017-12-01 09:21:06,664 handlers.py[line:104] ERROR NoneType

2017-12-01 09:23:08,086 handlers.py[line:104] ERROR NoneType

2017-12-01 09:24:08,727 handlers.py[line:104] ERROR NoneType

2017-12-01 09:26:10,092 handlers.py[line:104] ERROR NoneType

2017-12-01 09:27:10,773 handlers.py[line:104] ERROR NoneType

2017-12-01 09:29:12,166 handlers.py[line:104] ERROR NoneType

2017-12-01 09:30:12,858 handlers.py[line:104] ERROR NoneType

2017-12-01 09:30:15,614 handlers.py[line:104] ERROR NoneType

2017-12-01 09:30:15,616 handlers.py[line:104] ERROR NoneType

2017-12-01 09:30:15,617 handlers.py[line:104] ERROR NoneType

2017-12-01 09:30:15,618 handlers.py[line:104] ERROR NoneType

2017-12-01 09:30:15,619 handlers.py[line:104] ERROR NoneType

2017-12-01 09:31:13,708 handlers.py[line:104] ERROR NoneType

2017-12-01 09:32:14,363 handlers.py[line:104] ERROR NoneType

2017-12-01 09:34:15,743 handlers.py[line:104] ERROR NoneType

2017-12-01 09:35:16,447 handlers.py[line:104] ERROR NoneType

2017-12-01 09:36:17,109 handlers.py[line:104] ERROR NoneType

2017-12-01 09:37:17,782 handlers.py[line:104] ERROR NoneType

2017-12-01 09:38:18,456 handlers.py[line:104] ERROR NoneType

2017-12-01 09:40:19,853 handlers.py[line:104] ERROR NoneType

2017-12-01 09:44:22,898 handlers.py[line:104] ERROR NoneType

2017-12-01 09:44:22,901 handlers.py[line:104] ERROR NoneType

2017-12-01 09:45:23,540 handlers.py[line:104] ERROR NoneType

2017-12-01 09:46:24,240 handlers.py[line:104] ERROR NoneType

2017-12-01 09:46:24,242 handlers.py[line:104] ERROR NoneType

2017-12-01 09:49:26,642 handlers.py[line:104] ERROR NoneType

2017-12-01 09:50:27,317 handlers.py[line:104] ERROR NoneType

2017-12-01 09:50:27,319 handlers.py[line:104] ERROR NoneType

2017-12-01 09:51:28,112 handlers.py[line:104] ERROR NoneType

2017-12-01 09:52:28,904 handlers.py[line:104] ERROR NoneType

2017-12-01 09:52:28,906 handlers.py[line:104] ERROR NoneType

2017-12-01 09:53:29,629 handlers.py[line:104] ERROR NoneType

2017-12-01 09:54:30,419 handlers.py[line:104] ERROR NoneType

2017-12-01 09:55:31,239 handlers.py[line:104] ERROR NoneType

2017-12-01 09:56:22,985 handlers.py[line:104] ERROR NoneType

2017-12-01 09:56:33,060 handlers.py[line:104] ERROR NoneType

2017-12-01 09:57:23,292 handlers.py[line:104] ERROR NoneType

2017-12-01 09:58:24,008 handlers.py[line:104] ERROR NoneType

2017-12-01 09:58:25,320 handlers.py[line:104] ERROR NoneType

2017-12-01 09:58:34,968 handlers.py[line:104] ERROR NoneType

2017-12-01 09:59:24,733 handlers.py[line:104] ERROR NoneType

2017-12-01 09:59:24,735 handlers.py[line:104] ERROR NoneType

2017-12-01 09:59:35,151 handlers.py[line:104] ERROR NoneType

2017-12-01 09:59:35,156 handlers.py[line:104] ERROR NoneType

2017-12-01 10:00:25,547 handlers.py[line:104] ERROR NoneType

2017-12-01 10:01:26,232 handlers.py[line:104] ERROR NoneType

2017-12-01 10:01:26,233 handlers.py[line:104] ERROR NoneType

2017-12-01 10:02:26,949 handlers.py[line:104] ERROR NoneType

2017-12-01 10:03:38,376 handlers.py[line:104] ERROR NoneType

2017-12-01 10:04:28,330 handlers.py[line:104] ERROR NoneType

2017-12-01 10:04:28,332 handlers.py[line:104] ERROR NoneType

2017-12-01 10:10:01,935 handlers.py[line:104] ERROR NoneType

2017-12-01 10:10:01,937 handlers.py[line:104] ERROR NoneType

2017-12-01 10:12:02,965 handlers.py[line:104] ERROR NoneType

2017-12-01 10:14:04,549 handlers.py[line:104] ERROR NoneType

2017-12-01 10:15:05,204 handlers.py[line:104] ERROR NoneType

2017-12-01 10:16:05,903 handlers.py[line:104] ERROR NoneType

2017-12-01 10:17:06,570 handlers.py[line:104] ERROR NoneType

2017-12-01 10:18:07,224 handlers.py[line:104] ERROR NoneType

2017-12-01 10:19:07,895 handlers.py[line:104] ERROR NoneType

2017-12-01 10:20:08,608 handlers.py[line:104] ERROR NoneType

2017-12-01 10:22:10,070 handlers.py[line:104] ERROR NoneType

2017-12-01 10:23:10,766 handlers.py[line:104] ERROR NoneType

2017-12-01 10:23:10,769 handlers.py[line:104] ERROR NoneType

2017-12-01 10:24:11,448 handlers.py[line:104] ERROR NoneType

2017-12-01 10:24:11,449 handlers.py[line:104] ERROR NoneType

2017-12-01 10:25:12,117 handlers.py[line:104] ERROR NoneType

2017-12-01 10:25:12,119 handlers.py[line:104] ERROR NoneType

2017-12-01 10:26:12,849 handlers.py[line:104] ERROR NoneType

2017-12-01 10:26:12,853 handlers.py[line:104] ERROR NoneType

2017-12-01 10:27:13,511 handlers.py[line:104] ERROR NoneType

2017-12-01 10:28:14,184 handlers.py[line:104] ERROR NoneType

2017-12-01 10:30:15,842 handlers.py[line:104] ERROR NoneType

2017-12-01 10:31:16,494 handlers.py[line:104] ERROR NoneType

2017-12-01 10:32:17,423 handlers.py[line:104] ERROR NoneType

2017-12-01 10:32:17,424 handlers.py[line:104] ERROR NoneType

2017-12-01 10:33:18,152 handlers.py[line:104] ERROR NoneType

2017-12-01 10:34:18,885 handlers.py[line:104] ERROR NoneType

2017-12-01 10:34:18,888 handlers.py[line:104] ERROR NoneType

2017-12-01 10:36:20,256 handlers.py[line:104] ERROR NoneType

2017-12-01 10:37:21,017 handlers.py[line:104] ERROR NoneType

2017-12-01 10:38:21,815 handlers.py[line:104] ERROR NoneType

2017-12-01 10:39:22,502 handlers.py[line:104] ERROR NoneType

2017-12-01 10:40:23,176 handlers.py[line:104] ERROR NoneType

2017-12-01 10:42:24,613 handlers.py[line:104] ERROR NoneType

2017-12-01 10:42:24,616 handlers.py[line:104] ERROR NoneType

2017-12-01 10:43:25,308 handlers.py[line:104] ERROR NoneType

2017-12-01 10:44:26,146 handlers.py[line:104] ERROR NoneType

2017-12-01 10:45:26,829 handlers.py[line:104] ERROR NoneType

2017-12-01 10:46:27,664 handlers.py[line:104] ERROR NoneType

2017-12-01 10:47:28,361 handlers.py[line:104] ERROR NoneType

2017-12-01 10:48:29,047 handlers.py[line:104] ERROR NoneType

2017-12-01 10:49:30,006 handlers.py[line:104] ERROR NoneType

2017-12-01 10:50:30,723 handlers.py[line:104] ERROR NoneType

2017-12-01 10:51:31,728 handlers.py[line:104] ERROR NoneType

2017-12-01 10:51:31,730 handlers.py[line:104] ERROR NoneType

2017-12-01 10:51:31,730 handlers.py[line:104] ERROR NoneType

2017-12-01 10:53:32,993 handlers.py[line:104] ERROR NoneType

2017-12-01 10:53:33,003 handlers.py[line:104] ERROR NoneType

2017-12-01 10:55:34,391 handlers.py[line:104] ERROR NoneType

2017-12-01 11:49:31,137 handlers.py[line:104] ERROR NoneType

2017-12-01 11:50:31,583 handlers.py[line:104] ERROR NoneType

2017-12-01 11:51:32,659 handlers.py[line:104] ERROR NoneType

2017-12-01 11:51:32,661 handlers.py[line:104] ERROR NoneType

2017-12-01 11:52:33,325 handlers.py[line:104] ERROR NoneType

2017-12-01 11:53:34,109 handlers.py[line:104] ERROR NoneType

2017-12-01 11:56:36,444 handlers.py[line:104] ERROR NoneType

2017-12-01 11:57:37,119 handlers.py[line:104] ERROR NoneType

2017-12-01 11:57:37,120 handlers.py[line:104] ERROR NoneType

2017-12-01 11:58:37,813 handlers.py[line:104] ERROR NoneType

2017-12-01 11:59:38,466 handlers.py[line:104] ERROR NoneType

2017-12-01 12:01:39,868 handlers.py[line:104] ERROR NoneType

2017-12-01 12:02:40,767 handlers.py[line:104] ERROR NoneType

2017-12-01 12:02:40,771 handlers.py[line:104] ERROR NoneType

2017-12-01 12:04:42,138 handlers.py[line:104] ERROR NoneType

2017-12-01 12:05:42,905 handlers.py[line:104] ERROR NoneType

2017-12-01 12:05:42,907 handlers.py[line:104] ERROR NoneType

2017-12-01 12:07:44,231 handlers.py[line:104] ERROR NoneType

2017-12-01 12:08:44,890 handlers.py[line:104] ERROR NoneType

2017-12-01 12:08:44,898 handlers.py[line:104] ERROR NoneType

2017-12-01 12:10:46,257 handlers.py[line:104] ERROR NoneType

2017-12-01 12:10:46,259 handlers.py[line:104] ERROR NoneType

2017-12-01 12:13:48,433 handlers.py[line:104] ERROR NoneType

2017-12-01 12:13:48,436 handlers.py[line:104] ERROR NoneType

2017-12-01 12:14:50,098 handlers.py[line:104] ERROR NoneType

2017-12-01 12:17:51,613 handlers.py[line:104] ERROR NoneType

2017-12-01 12:17:51,614 handlers.py[line:104] ERROR NoneType

2017-12-01 12:19:53,972 handlers.py[line:104] ERROR NoneType

2017-12-01 12:19:53,985 handlers.py[line:104] ERROR NoneType

2017-12-01 12:20:54,081 handlers.py[line:104] ERROR NoneType

2017-12-01 12:21:54,776 handlers.py[line:104] ERROR NoneType

2017-12-01 12:23:56,197 handlers.py[line:104] ERROR NoneType

2017-12-01 12:24:56,963 handlers.py[line:104] ERROR NoneType

2017-12-01 12:27:59,519 handlers.py[line:104] ERROR NoneType

2017-12-01 12:31:01,572 handlers.py[line:104] ERROR NoneType

2017-12-01 12:31:01,573 handlers.py[line:104] ERROR NoneType

2017-12-01 12:33:02,958 handlers.py[line:104] ERROR NoneType

2017-12-01 12:34:03,667 handlers.py[line:104] ERROR NoneType

2017-12-01 12:34:03,668 handlers.py[line:104] ERROR NoneType

2017-12-01 12:36:05,042 handlers.py[line:104] ERROR NoneType

2017-12-01 12:39:07,112 handlers.py[line:104] ERROR NoneType

2017-12-01 12:39:07,114 handlers.py[line:104] ERROR NoneType

2017-12-01 12:42:09,235 handlers.py[line:104] ERROR NoneType

2017-12-01 12:44:10,662 handlers.py[line:104] ERROR NoneType

2017-12-01 12:45:11,363 handlers.py[line:104] ERROR NoneType

2017-12-01 12:47:12,773 handlers.py[line:104] ERROR NoneType

2017-12-01 12:53:17,090 handlers.py[line:104] ERROR NoneType

2017-12-01 12:53:17,092 handlers.py[line:104] ERROR NoneType

2017-12-01 12:55:18,490 handlers.py[line:104] ERROR NoneType

2017-12-01 12:56:19,189 handlers.py[line:104] ERROR NoneType

2017-12-01 13:01:22,728 handlers.py[line:104] ERROR NoneType

2017-12-01 13:01:22,729 handlers.py[line:104] ERROR NoneType

2017-12-01 13:04:24,812 handlers.py[line:104] ERROR NoneType

2017-12-01 13:06:26,166 handlers.py[line:104] ERROR NoneType

2017-12-01 13:09:28,274 handlers.py[line:104] ERROR NoneType

2017-12-01 13:10:29,924 handlers.py[line:104] ERROR NoneType

2017-12-01 13:12:30,711 handlers.py[line:104] ERROR NoneType

2017-12-01 13:13:31,387 handlers.py[line:104] ERROR NoneType

2017-12-01 13:15:32,758 handlers.py[line:104] ERROR NoneType

2017-12-01 13:15:32,760 handlers.py[line:104] ERROR NoneType

2017-12-01 13:15:32,761 handlers.py[line:104] ERROR NoneType

2017-12-01 13:18:34,847 handlers.py[line:104] ERROR NoneType

2017-12-01 13:21:37,199 handlers.py[line:104] ERROR NoneType

2017-12-01 13:21:37,201 handlers.py[line:104] ERROR NoneType

2017-12-01 13:24:39,298 handlers.py[line:104] ERROR NoneType

2017-12-01 13:24:39,299 handlers.py[line:104] ERROR NoneType

2017-12-01 13:28:42,496 handlers.py[line:104] ERROR NoneType

2017-12-01 13:28:42,498 handlers.py[line:104] ERROR NoneType

2017-12-01 13:28:42,499 handlers.py[line:104] ERROR NoneType

2017-12-01 13:30:43,958 handlers.py[line:104] ERROR NoneType

2017-12-01 13:31:44,663 handlers.py[line:104] ERROR NoneType

2017-12-01 13:33:46,129 handlers.py[line:104] ERROR NoneType

2017-12-01 13:33:46,137 handlers.py[line:104] ERROR NoneType

2017-12-01 13:35:47,628 handlers.py[line:104] ERROR NoneType

2017-12-01 13:36:48,363 handlers.py[line:104] ERROR NoneType

2017-12-01 13:37:49,070 handlers.py[line:104] ERROR NoneType

2017-12-01 13:37:49,074 handlers.py[line:104] ERROR NoneType

2017-12-01 13:38:49,808 handlers.py[line:104] ERROR NoneType

2017-12-01 13:38:49,810 handlers.py[line:104] ERROR NoneType

2017-12-01 13:39:50,598 handlers.py[line:104] ERROR NoneType

2017-12-01 13:40:51,340 handlers.py[line:104] ERROR NoneType

2017-12-01 13:41:52,469 handlers.py[line:104] ERROR NoneType

2017-12-01 14:52:34,399 handlers.py[line:104] ERROR NoneType

2017-12-01 14:54:17,790 handlers.py[line:104] ERROR NoneType

2017-12-01 14:54:17,791 handlers.py[line:104] ERROR NoneType

2017-12-01 14:55:59,611 handlers.py[line:104] ERROR NoneType

2017-12-01 14:55:59,613 handlers.py[line:104] ERROR NoneType

2017-12-01 15:17:38,654 handlers.py[line:104] ERROR NoneType

2017-12-01 15:17:57,607 handlers.py[line:104] ERROR NoneType

2017-12-01 15:20:59,242 handlers.py[line:104] ERROR NoneType

2017-12-01 15:21:59,941 handlers.py[line:104] ERROR NoneType

2017-12-01 15:24:01,395 handlers.py[line:104] ERROR NoneType

2017-12-01 15:25:02,098 handlers.py[line:104] ERROR NoneType

2017-12-01 15:26:03,010 handlers.py[line:104] ERROR NoneType

2017-12-01 15:27:03,686 handlers.py[line:104] ERROR NoneType

2017-12-01 15:33:55,082 handlers.py[line:104] ERROR NoneType

2017-12-01 15:34:55,915 handlers.py[line:104] ERROR NoneType

2017-12-01 15:35:56,287 handlers.py[line:104] ERROR NoneType

2017-12-01 15:36:57,073 handlers.py[line:104] ERROR NoneType

2017-12-01 15:37:23,879 handlers.py[line:104] ERROR NoneType

2017-12-01 15:37:23,883 handlers.py[line:104] ERROR NoneType

2017-12-01 15:37:23,884 handlers.py[line:104] ERROR NoneType

2017-12-01 15:38:42,670 handlers.py[line:104] ERROR NoneType

2017-12-01 15:38:42,678 handlers.py[line:104] ERROR NoneType

2017-12-01 15:38:42,679 handlers.py[line:104] ERROR NoneType

2017-12-01 15:39:11,289 handlers.py[line:104] ERROR NoneType

2017-12-01 15:39:42,858 handlers.py[line:104] ERROR NoneType

2017-12-01 15:39:42,983 handlers.py[line:104] ERROR NoneType

2017-12-01 15:41:12,655 handlers.py[line:104] ERROR NoneType

2017-12-01 15:42:52,888 handlers.py[line:104] ERROR NoneType

2017-12-01 15:42:52,889 handlers.py[line:104] ERROR NoneType

2017-12-01 15:42:52,891 handlers.py[line:104] ERROR NoneType

2017-12-01 15:42:52,892 handlers.py[line:104] ERROR NoneType

2017-12-01 15:43:24,303 handlers.py[line:104] ERROR NoneType

2017-12-01 15:43:53,025 handlers.py[line:104] ERROR NoneType

2017-12-01 15:43:53,028 handlers.py[line:104] ERROR NoneType

2017-12-01 15:43:53,192 handlers.py[line:104] ERROR NoneType

2017-12-01 15:45:25,706 handlers.py[line:104] ERROR NoneType

2017-12-01 15:46:26,450 handlers.py[line:104] ERROR NoneType

2017-12-01 15:47:27,108 handlers.py[line:104] ERROR NoneType

2017-12-01 15:49:28,441 handlers.py[line:104] ERROR NoneType

2017-12-01 15:50:29,194 handlers.py[line:104] ERROR NoneType

2017-12-01 15:51:29,893 handlers.py[line:104] ERROR NoneType

2017-12-01 15:53:31,195 handlers.py[line:104] ERROR NoneType

2017-12-01 15:53:31,198 handlers.py[line:104] ERROR NoneType

2017-12-01 15:54:57,144 handlers.py[line:104] ERROR NoneType

2017-12-01 15:55:32,533 handlers.py[line:104] ERROR NoneType

2017-12-01 15:55:32,543 handlers.py[line:104] ERROR NoneType

2017-12-01 15:56:33,262 handlers.py[line:104] ERROR NoneType

2017-12-01 15:57:33,932 handlers.py[line:104] ERROR NoneType

2017-12-01 15:58:34,621 handlers.py[line:104] ERROR NoneType

2017-12-01 15:58:34,625 handlers.py[line:104] ERROR NoneType

2017-12-01 15:59:35,293 handlers.py[line:104] ERROR NoneType

2017-12-01 16:00:35,927 handlers.py[line:104] ERROR NoneType

2017-12-01 16:03:02,504 handlers.py[line:104] ERROR NoneType

2017-12-01 16:03:38,219 handlers.py[line:104] ERROR NoneType

2017-12-01 16:03:38,220 handlers.py[line:104] ERROR NoneType

2017-12-01 16:04:38,877 handlers.py[line:104] ERROR NoneType

2017-12-01 16:05:39,565 handlers.py[line:104] ERROR NoneType

2017-12-01 16:05:39,566 handlers.py[line:104] ERROR NoneType

2017-12-01 16:07:00,040 handlers.py[line:104] ERROR NoneType

2017-12-01 16:07:40,953 handlers.py[line:104] ERROR NoneType

2017-12-01 16:07:40,955 handlers.py[line:104] ERROR NoneType

2017-12-01 16:08:41,660 handlers.py[line:104] ERROR NoneType

2017-12-01 16:09:42,307 handlers.py[line:104] ERROR NoneType

2017-12-01 16:10:42,989 handlers.py[line:104] ERROR NoneType

2017-12-01 16:11:43,641 handlers.py[line:104] ERROR NoneType

2017-12-01 16:11:43,643 handlers.py[line:104] ERROR NoneType

2017-12-01 16:12:44,319 handlers.py[line:104] ERROR NoneType

2017-12-01 16:13:46,051 handlers.py[line:104] ERROR NoneType

2017-12-01 16:14:46,198 handlers.py[line:104] ERROR NoneType

2017-12-01 16:15:47,024 handlers.py[line:104] ERROR NoneType

2017-12-01 16:15:47,028 handlers.py[line:104] ERROR NoneType

2017-12-01 16:20:17,435 handlers.py[line:104] ERROR NoneType

2017-12-01 16:21:09,892 handlers.py[line:104] ERROR NoneType

2017-12-01 16:21:09,896 handlers.py[line:104] ERROR NoneType

2017-12-01 16:21:09,911 handlers.py[line:104] ERROR NoneType

2017-12-01 16:21:16,435 handlers.py[line:104] ERROR NoneType

2017-12-01 16:22:08,760 handlers.py[line:104] ERROR NoneType

2017-12-01 16:22:08,761 handlers.py[line:104] ERROR NoneType

2017-12-01 16:22:09,464 handlers.py[line:104] ERROR NoneType

2017-12-01 16:22:10,209 handlers.py[line:104] ERROR NoneType

2017-12-01 16:22:10,213 handlers.py[line:104] ERROR NoneType

2017-12-01 16:22:11,001 handlers.py[line:104] ERROR NoneType

2017-12-01 16:22:45,152 handlers.py[line:104] ERROR NoneType

2017-12-01 16:23:45,828 handlers.py[line:104] ERROR NoneType

2017-12-01 16:24:46,487 handlers.py[line:104] ERROR NoneType

2017-12-01 16:25:47,163 handlers.py[line:104] ERROR NoneType

2017-12-01 16:25:47,164 handlers.py[line:104] ERROR NoneType

2017-12-01 16:26:16,120 handlers.py[line:104] ERROR NoneType

2017-12-01 16:26:16,184 handlers.py[line:104] ERROR NoneType

2017-12-01 16:26:47,861 handlers.py[line:104] ERROR NoneType

2017-12-01 16:27:48,504 handlers.py[line:104] ERROR NoneType

2017-12-01 16:28:49,432 handlers.py[line:104] ERROR NoneType

2017-12-01 16:29:14,315 handlers.py[line:104] ERROR NoneType

2017-12-01 16:29:37,025 handlers.py[line:104] ERROR NoneType

2017-12-01 16:29:37,027 handlers.py[line:104] ERROR NoneType

2017-12-01 16:29:37,029 handlers.py[line:104] ERROR NoneType

2017-12-01 16:29:37,030 handlers.py[line:104] ERROR NoneType

2017-12-01 16:29:37,031 handlers.py[line:104] ERROR NoneType

2017-12-01 16:29:37,032 handlers.py[line:104] ERROR NoneType

2017-12-01 16:29:50,149 handlers.py[line:104] ERROR NoneType

2017-12-01 16:30:36,926 handlers.py[line:104] ERROR NoneType

2017-12-01 16:30:36,928 handlers.py[line:104] ERROR NoneType

2017-12-01 16:30:37,161 handlers.py[line:104] ERROR NoneType

2017-12-01 16:30:50,892 handlers.py[line:104] ERROR NoneType

2017-12-01 16:31:51,607 handlers.py[line:104] ERROR NoneType

2017-12-01 16:32:39,900 handlers.py[line:104] ERROR NoneType

2017-12-01 16:32:52,271 handlers.py[line:104] ERROR NoneType

2017-12-01 16:33:52,911 handlers.py[line:104] ERROR NoneType

2017-12-01 16:34:53,654 handlers.py[line:104] ERROR NoneType

2017-12-01 16:34:54,586 handlers.py[line:104] ERROR NoneType

2017-12-01 16:34:56,808 handlers.py[line:104] ERROR NoneType

2017-12-01 16:34:56,810 handlers.py[line:104] ERROR NoneType

2017-12-01 16:34:56,811 handlers.py[line:104] ERROR NoneType

2017-12-01 16:34:56,812 handlers.py[line:104] ERROR NoneType

2017-12-01 16:34:56,816 handlers.py[line:104] ERROR NoneType

2017-12-01 16:36:55,775 handlers.py[line:104] ERROR NoneType

2017-12-01 16:37:56,855 handlers.py[line:104] ERROR NoneType

2017-12-01 16:38:57,565 handlers.py[line:104] ERROR NoneType

2017-12-01 16:38:57,567 handlers.py[line:104] ERROR NoneType

2017-12-01 16:39:58,309 handlers.py[line:104] ERROR NoneType

2017-12-01 16:39:58,310 handlers.py[line:104] ERROR NoneType

2017-12-01 16:40:25,887 handlers.py[line:104] ERROR NoneType

2017-12-01 16:40:59,383 handlers.py[line:104] ERROR NoneType

2017-12-01 16:41:26,860 handlers.py[line:104] ERROR NoneType

2017-12-01 16:42:00,398 handlers.py[line:104] ERROR NoneType

2017-12-01 16:43:01,107 handlers.py[line:104] ERROR NoneType

2017-12-01 16:43:28,861 handlers.py[line:104] ERROR NoneType

2017-12-01 16:44:02,041 handlers.py[line:104] ERROR NoneType

2017-12-01 16:44:29,896 handlers.py[line:104] ERROR NoneType

2017-12-01 16:46:03,672 handlers.py[line:104] ERROR NoneType

2017-12-01 16:47:43,829 handlers.py[line:104] ERROR NoneType

2017-12-01 16:48:05,149 handlers.py[line:104] ERROR NoneType

2017-12-01 16:50:46,822 handlers.py[line:104] ERROR NoneType

2017-12-01 16:51:27,038 handlers.py[line:104] ERROR NoneType

2017-12-01 16:52:47,943 handlers.py[line:104] ERROR NoneType

2017-12-01 16:53:54,738 handlers.py[line:104] ERROR NoneType

2017-12-01 16:54:55,626 handlers.py[line:104] ERROR NoneType

2017-12-01 16:55:50,903 handlers.py[line:104] ERROR NoneType

2017-12-01 16:56:48,530 handlers.py[line:104] ERROR NoneType

2017-12-01 16:57:07,495 handlers.py[line:104] ERROR NoneType

2017-12-01 16:57:52,903 handlers.py[line:104] ERROR NoneType

2017-12-01 16:58:55,182 handlers.py[line:104] ERROR NoneType

2017-12-01 16:58:58,827 handlers.py[line:104] ERROR NoneType

2017-12-01 16:59:58,756 handlers.py[line:104] ERROR NoneType

2017-12-01 17:01:21,123 handlers.py[line:104] ERROR NoneType

2017-12-01 17:01:26,245 handlers.py[line:104] ERROR NoneType

2017-12-01 17:02:00,840 handlers.py[line:104] ERROR NoneType

2017-12-01 17:02:21,862 handlers.py[line:104] ERROR NoneType

2017-12-01 17:03:22,518 handlers.py[line:104] ERROR NoneType

2017-12-01 17:04:05,496 handlers.py[line:104] ERROR NoneType

2017-12-01 17:05:03,824 handlers.py[line:104] ERROR NoneType

2017-12-01 17:06:15,623 handlers.py[line:104] ERROR NoneType

2017-12-01 17:06:38,790 handlers.py[line:104] ERROR NoneType

2017-12-01 17:06:39,797 handlers.py[line:104] ERROR NoneType

2017-12-01 17:07:17,847 handlers.py[line:104] ERROR NoneType

2017-12-01 17:08:40,647 handlers.py[line:104] ERROR NoneType

2017-12-01 17:09:19,836 handlers.py[line:104] ERROR NoneType

2017-12-01 17:09:41,655 handlers.py[line:104] ERROR NoneType

2017-12-01 17:10:20,912 handlers.py[line:104] ERROR NoneType

2017-12-01 17:11:43,016 handlers.py[line:104] ERROR NoneType

2017-12-01 17:12:16,720 handlers.py[line:104] ERROR NoneType

2017-12-01 17:13:44,399 handlers.py[line:104] ERROR NoneType

2017-12-01 17:14:25,847 handlers.py[line:104] ERROR NoneType

2017-12-01 17:15:55,803 handlers.py[line:104] ERROR NoneType

2017-12-01 17:15:56,732 handlers.py[line:104] ERROR NoneType

2017-12-01 17:16:56,773 handlers.py[line:104] ERROR NoneType

2017-12-01 17:17:28,894 handlers.py[line:104] ERROR NoneType

2017-12-01 17:17:28,897 handlers.py[line:104] ERROR Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/sanic/app.py", line 453, in handle_request
    response = await self._run_request_middleware(request)
  File "/usr/lib/python3.5/asyncio/coroutines.py", line 124, in throw
    return self.gen.throw(exc)
  File "/usr/local/lib/python3.5/dist-packages/sanic/app.py", line 611, in _run_request_middleware
    response = await response
  File "/usr/lib/python3.5/asyncio/coroutines.py", line 124, in throw
    return self.gen.throw(exc)
  File "main_server.py", line 1174, in add_session_to_request
    await session_interface.open(request)
  File "/usr/lib/python3.5/asyncio/coroutines.py", line 124, in throw
    return self.gen.throw(exc)
  File "/usr/local/lib/python3.5/dist-packages/sanic_session/redis_session_interface.py", line 61, in open
    val = await redis_connection.get(self.prefix + sid)
  File "/usr/local/lib/python3.5/dist-packages/asyncio_redis/protocol.py", line 676, in wrapper
    result = yield from method(protocol_self, _NoTransaction, *a, **kw)
  File "/usr/local/lib/python3.5/dist-packages/asyncio_redis/protocol.py", line 1135, in _query
    result = yield from self._get_answer(transaction, answer_f, _bypass=_bypass, call=call)
  File "/usr/local/lib/python3.5/dist-packages/asyncio_redis/protocol.py", line 1062, in _get_answer
    result = yield from answer_f
  File "/usr/lib/python3.5/asyncio/futures.py", line 361, in __iter__
    yield self  # This tells Task to wait for completion.
  File "uvloop/future.pyx", line 434, in uvloop.loop.BaseTask._fast_wakeup (uvloop/loop.c:114015)
  File "/usr/lib/python3.5/asyncio/futures.py", line 266, in result
    raise CancelledError
concurrent.futures._base.CancelledError

2017-12-01 17:17:28,898 handlers.py[line:120] ERROR Exception occurred while handling uri: "http://39.108.67.178:5000/company/user"
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/sanic/app.py", line 453, in handle_request
    response = await self._run_request_middleware(request)
  File "/usr/lib/python3.5/asyncio/coroutines.py", line 124, in throw
    return self.gen.throw(exc)
  File "/usr/local/lib/python3.5/dist-packages/sanic/app.py", line 611, in _run_request_middleware
    response = await response
  File "/usr/lib/python3.5/asyncio/coroutines.py", line 124, in throw
    return self.gen.throw(exc)
  File "main_server.py", line 1174, in add_session_to_request
    await session_interface.open(request)
  File "/usr/lib/python3.5/asyncio/coroutines.py", line 124, in throw
    return self.gen.throw(exc)
  File "/usr/local/lib/python3.5/dist-packages/sanic_session/redis_session_interface.py", line 61, in open
    val = await redis_connection.get(self.prefix + sid)
  File "/usr/local/lib/python3.5/dist-packages/asyncio_redis/protocol.py", line 676, in wrapper
    result = yield from method(protocol_self, _NoTransaction, *a, **kw)
  File "/usr/local/lib/python3.5/dist-packages/asyncio_redis/protocol.py", line 1135, in _query
    result = yield from self._get_answer(transaction, answer_f, _bypass=_bypass, call=call)
  File "/usr/local/lib/python3.5/dist-packages/asyncio_redis/protocol.py", line 1062, in _get_answer
    result = yield from answer_f
  File "/usr/lib/python3.5/asyncio/futures.py", line 361, in __iter__
    yield self  # This tells Task to wait for completion.
  File "uvloop/future.pyx", line 434, in uvloop.loop.BaseTask._fast_wakeup (uvloop/loop.c:114015)
  File "/usr/lib/python3.5/asyncio/futures.py", line 266, in result
    raise CancelledError
concurrent.futures._base.CancelledError

2017-12-01 17:17:28,900 server.py[line:217] ERROR Connection lost before response written @ ('180.173.40.150', 55756)
2017-12-01 17:18:05,078 handlers.py[line:104] ERROR NoneType

2017-12-01 17:18:06,000 handlers.py[line:104] ERROR NoneType

2017-12-01 17:19:06,383 handlers.py[line:104] ERROR NoneType

2017-12-01 17:20:07,059 handlers.py[line:104] ERROR NoneType

2017-12-01 17:21:07,737 handlers.py[line:104] ERROR NoneType

2017-12-01 17:22:08,626 handlers.py[line:104] ERROR NoneType

2017-12-01 17:22:32,942 handlers.py[line:104] ERROR NoneType

2017-12-01 17:23:09,276 handlers.py[line:104] ERROR NoneType

2017-12-01 17:23:33,944 handlers.py[line:104] ERROR NoneType

2017-12-01 17:24:09,968 handlers.py[line:104] ERROR NoneType

2017-12-01 17:24:34,956 handlers.py[line:104] ERROR NoneType

2017-12-01 17:25:11,618 handlers.py[line:104] ERROR NoneType

2017-12-01 17:25:11,621 handlers.py[line:104] ERROR NoneType

2017-12-01 17:26:11,704 handlers.py[line:104] ERROR NoneType

2017-12-01 17:26:36,942 handlers.py[line:104] ERROR NoneType

2017-12-01 17:27:12,407 handlers.py[line:104] ERROR NoneType

2017-12-01 17:27:37,927 handlers.py[line:104] ERROR NoneType

2017-12-01 17:28:13,075 handlers.py[line:104] ERROR NoneType

2017-12-01 17:29:13,700 handlers.py[line:104] ERROR NoneType

2017-12-01 17:29:40,935 handlers.py[line:104] ERROR NoneType

2017-12-01 17:29:40,936 app.py[line:498] ERROR Exception occured in one of response middleware handlers
Traceback (most recent call last):
  File "/usr/local/lib/python3.5/dist-packages/sanic/app.py", line 495, in handle_request
    response)
  File "/usr/lib/python3.5/asyncio/coroutines.py", line 124, in throw
    return self.gen.throw(exc)
  File "/usr/local/lib/python3.5/dist-packages/sanic/app.py", line 621, in _run_response_middleware
    _response = await _response
  File "/usr/lib/python3.5/asyncio/coroutines.py", line 124, in throw
    return self.gen.throw(exc)
  File "main_server.py", line 1181, in save_session
    await session_interface.save(request, response)
  File "/usr/lib/python3.5/asyncio/coroutines.py", line 124, in throw
    return self.gen.throw(exc)
  File "/usr/local/lib/python3.5/dist-packages/sanic_session/redis_session_interface.py", line 100, in save
    await redis_connection.setex(key, self.expiry, val)
  File "/usr/local/lib/python3.5/dist-packages/asyncio_redis/protocol.py", line 676, in wrapper
    result = yield from method(protocol_self, _NoTransaction, *a, **kw)
  File "/usr/local/lib/python3.5/dist-packages/asyncio_redis/protocol.py", line 1135, in _query
    result = yield from self._get_answer(transaction, answer_f, _bypass=_bypass, call=call)
  File "/usr/local/lib/python3.5/dist-packages/asyncio_redis/protocol.py", line 1062, in _get_answer
    result = yield from answer_f
  File "/usr/lib/python3.5/asyncio/futures.py", line 361, in __iter__
    yield self  # This tells Task to wait for completion.
  File "uvloop/future.pyx", line 434, in uvloop.loop.BaseTask._fast_wakeup (uvloop/loop.c:114015)
  File "/usr/lib/python3.5/asyncio/futures.py", line 266, in result
    raise CancelledError
concurrent.futures._base.CancelledError
2017-12-01 17:31:15,224 handlers.py[line:104] ERROR NoneType

2017-12-01 17:32:15,863 handlers.py[line:104] ERROR NoneType

2017-12-01 17:32:19,813 handlers.py[line:104] ERROR NoneType

2017-12-01 17:32:19,815 handlers.py[line:104] ERROR NoneType

2017-12-01 17:32:19,816 handlers.py[line:104] ERROR NoneType

2017-12-01 17:32:19,817 handlers.py[line:104] ERROR NoneType

2017-12-01 17:32:19,818 handlers.py[line:104] ERROR NoneType

2017-12-01 17:34:00,608 handlers.py[line:104] ERROR NoneType

2017-12-01 17:34:37,920 handlers.py[line:104] ERROR NoneType

2017-12-01 17:35:01,425 handlers.py[line:104] ERROR NoneType

2017-12-01 17:36:02,112 handlers.py[line:104] ERROR NoneType

2017-12-01 17:37:02,840 handlers.py[line:104] ERROR NoneType

2017-12-01 17:37:02,841 handlers.py[line:104] ERROR NoneType

2017-12-01 17:37:38,954 handlers.py[line:104] ERROR NoneType

2017-12-01 17:38:03,504 handlers.py[line:104] ERROR NoneType

2017-12-01 17:39:04,378 handlers.py[line:104] ERROR NoneType

2017-12-01 17:40:05,040 handlers.py[line:104] ERROR NoneType

2017-12-01 17:40:05,041 handlers.py[line:104] ERROR NoneType

2017-12-01 17:41:05,694 handlers.py[line:104] ERROR NoneType

2017-12-01 17:42:06,424 handlers.py[line:104] ERROR NoneType

2017-12-01 17:43:07,068 handlers.py[line:104] ERROR NoneType

2017-12-01 17:43:41,444 handlers.py[line:104] ERROR NoneType

2017-12-01 17:44:07,745 handlers.py[line:104] ERROR NoneType

2017-12-01 17:45:08,400 handlers.py[line:104] ERROR NoneType

2017-12-01 17:46:09,061 handlers.py[line:104] ERROR NoneType

2017-12-01 17:46:43,876 handlers.py[line:104] ERROR NoneType

2017-12-01 17:48:10,360 handlers.py[line:104] ERROR NoneType

2017-12-01 17:48:10,362 handlers.py[line:104] ERROR NoneType

2017-12-01 17:48:10,363 handlers.py[line:104] ERROR NoneType

2017-12-01 17:48:45,893 handlers.py[line:104] ERROR NoneType

2017-12-01 17:49:46,265 handlers.py[line:104] ERROR NoneType

2017-12-01 17:50:12,345 handlers.py[line:104] ERROR NoneType

2017-12-01 17:50:12,549 handlers.py[line:104] ERROR NoneType

2017-12-01 17:50:12,550 handlers.py[line:104] ERROR NoneType

2017-12-01 17:50:51,660 handlers.py[line:104] ERROR NoneType

2017-12-01 17:51:15,684 handlers.py[line:104] ERROR NoneType

2017-12-01 17:52:29,010 handlers.py[line:104] ERROR NoneType

2017-12-01 17:53:37,878 handlers.py[line:104] ERROR NoneType

2017-12-01 17:55:15,695 handlers.py[line:104] ERROR NoneType

